ArcadiaTeamsBot/Dialogs/NewRequestDialog.cs

```diff
@@ -6,42 +6,30 @@
 
     using Microsoft.Bot.Builder;
     using Microsoft.Bot.Builder.Dialogs;
-    using Microsoft.Bot.Builder.Dialogs.Choices;
+    using Microsoft.Bot.Schema;
+
+    using Cards;
 
     public class NewRequestDialog : ComponentDialog
     {
         public NewRequestDialog() : base(nameof(NewRequestDialog))
         {
             AddDialog(new WaterfallDialog(nameof(WaterfallDialog), new WaterfallStep[]
             {
-                TypeStep,
-                AnotherSteps,
+                InfoStep,
             }));
 
             AddDialog(new TextPrompt(nameof(TextPrompt)));
-            AddDialog(new ChoicePrompt(nameof(ChoicePrompt)));
             InitialDialogId = nameof(WaterfallDialog);
         }
 
-        private static async Task<DialogTurnResult> TypeStep(WaterfallStepContext stepContext, CancellationToken cancellationToken)
-        {
-            return await stepContext.PromptAsync(nameof(ChoicePrompt),
-                new PromptOptions
-                {
-                    Prompt = MessageFactory.Text("What request do you want to create?"),
-                    //types from serviceDesk
-                    Choices = ChoiceFactory.ToChoices(new List<string> { "transport", "ndfl" }),
-                }, cancellationToken);
-        }
-
-        private static async Task<DialogTurnResult> AnotherSteps(WaterfallStepContext stepContext, CancellationToken cancellationToken)
+        private static async Task<DialogTurnResult> InfoStep(WaterfallStepContext stepContext, CancellationToken cancellationToken)
         {
-            stepContext.Values["Type"] = ((FoundChoice)stepContext.Result).Value;
-            await stepContext.PromptAsync(nameof(TextPrompt),
-                new PromptOptions
-                {
-                    Prompt = MessageFactory.Text("another questions "),
-                }, cancellationToken);
+            var attachments = new List<Attachment>();
+            var reply = MessageFactory.Attachment(attachments);
+            reply.Attachments.Add(Cards.GetInfoCard().ToAttachment());
+            await stepContext.Context.SendActivityAsync(reply, cancellationToken);
+            await stepContext.Context.SendActivityAsync(MessageFactory.Text("Type anything to continue."), cancellationToken);
```

Comment:
To many code lines in a row without blank lines between. Looks messy.

Reply:
Added lines